packages/page-council/src/Overview/Vote.tsx

```diff
@@ -109,6 +109,19 @@ function Vote ({ electionsInfo }: Props): React.ReactElement<Props> {
             </Modal.Columns>
           </Modal.Content>
           <Modal.Actions onCancel={toggleVisible}>
+            <TxButton
+              accountId={accountId}
+              icon='trash-alt'
+              isDisabled={!defaultVotes.length}
+              label={t<string>('Clear all')}
```

Comment:
```suggestion
              label={t<string>('Unvote prior')}
```

Tried "clear prior", "remove prior" - what about the above? (It current be confused with clearing the selection above, hence the rename). Naming stuff is hard.

Reply:
Because the selection can change (and prior might be confusing), what do you think about 
```suggestion
              label={t<string>('Unvote all')}
```